meetings/2024-01-31.md

```diff
@@ -41,28 +49,95 @@
+* SC: Again, if it's CG's work item, I would consider that as a topic on agenda. If we leave it all the way to the end I am not sure if we'll be able to address all the point. This group was as fast as it could've in processing the issues and PRs in the past. I trust PAC's judgement, but I still think the CG should be plugged in.
+* RG: is the group willing to do a Special Actions Meeting with PAC leading to resolve this.
+* PAC: by asking this question I didn't want to seem like I am blaming the CG for being slow. Again, I welcome feedback. If we need an extensive discussion on some points, I'll try to attend these meetings.
+* VB: Is the primary concern that our meeting frequency will delay the submission?
+* PAC: Yes. Ideally I'd like to merge a PR within a few days. It's not a criticism on the CG. It's just that I want to have a fast pace for the next iteration, but not bypassing the CG.
+* VB: Let's take advantage of the async work, follow repo, make sure people are aware of PRs. If we need ad-hoc meetings outside the regular one we can have them.
+* SC: No strong opinion on the time. We don't need to wait for 2 weeks for a typo to be fixed, we work in an open space, other things may need more feedback. See https://github.com/w3c-cg/solid/blob/main/CONTRIBUTING.md#decisions as example.
+* eP: The charter was proposed by the W3C team, not the CG. PAC is doing a best effort. I think we should not do anything that would slow PAC down. If we're all happy with the way it's done, we don't need to prematurely optimize. I don't think anyone is concerned that feedback from CG will be ignored. If someone wants to get something done, we shouldn't put too much procedure around it.
+* VB: is there an objection to working async and let PAC do his work. Any objections? (NONE from audience).
+
+### Access modes discussion
+URL: https://hackmd.io/@solid/ByPmZZ4KT
 
-### Special Topic Meetings
-* URL: https://github.com/orgs/solid/projects/16/views/1
-
-#### Access modes discussion
 * Proposed by eP
-URL: https://hackmd.io/@solid/ByPmZZ4KT
+* VB: There are a number of open questions. Specs need to be reviewed. How do you think we could move forward with this? SC, eP?
+* eP: I don't think we can make progress during this meeting. The question is where should be tracking that. Maybe we could try over the next week or two to make progress.
+* VB: any comments on that? In case we need a separate meeting we can schedule any time.
+* SC: I think if we try to integrate the concerns in those issues into respective documents, it'd reduce the things we'll need to address separately. I don't think we need a separate place to track it since we have all the issues. Let's say we need to update the definition of operations, we know these things. If this is in the scope of the spec we'll clear it up. We have the information already.
```

Comment:
```suggestion
* VB: any comments on that? If we need a separate meeting, we can schedule any time.
* SC: I think that if we try to integrate the concerns in those issues into respective documents, it'd reduce the things we'll need to address separately. I don't think we need a separate place to track it since we have all the issues. Let's say we need to update the definition of operations; we know these things. If this is in the scope of the spec we'll clear it up. We have the information already.
```